PollBuddy-Server/frontend/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Router, Link } from "@reach/router";
+import { Router } from "@reach/router";
```

Comment:
Why was link removed? Do we need it?

Reply:
This removed a warning from the console. I found no changes in any functionalities